pkg/operator/controllers/gameserverbuild_controller.go

```diff
@@ -63,6 +63,11 @@ const (
 	maxNumberOfGameServersToDelete = 20
 )
 
+type MutexMap struct {
```

Comment:
can we add a comment that it's used for the creations/deletions expectations code?

Reply:
Done!